app/main.py

```diff
@@ -1,13 +1,31 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    def format_linter_error(error):
```

Comment:
The nested function definition `def format_linter_error(error):` is redundant. The outer function definition already serves the purpose. Consider removing this line.